packages/ts/react-crud/src/property-info.ts

```diff
@@ -59,34 +60,47 @@ export function _generateHeader(path: string): string {
     .replace(/^./u, (match) => match.toUpperCase());
 }
 
-export const getPropertyIds = (model: DetachedModelConstructor<AbstractModel>): string[] =>
-  Object.keys(Object.getOwnPropertyDescriptors(model.prototype)).filter((p) => p !== 'constructor');
+export const getPropertyIds = (model: DetachedModelConstructor<AbstractModel>): string[] => {
+  const propertyIds: string[] = [];
+
+  for (let proto = model; proto !== ObjectModel; proto = Object.getPrototypeOf(proto)) {
+    // the choice of putting parent properties at the beginning is deliberate, `push` could be used instead
```

Comment:
Explaining what the alternatives could be is unnecessary and just adds cognitive load. Better to just explain what it does.

Reply:
```suggestion
    // parent properties are added at the beginning
```

Reply:
Looks good 👍